contracts/truefi2/FixedTermLoanAgency.sol

```diff
@@ -403,29 +403,21 @@ contract FixedTermLoanAgency is IFixedTermLoanAgency, UpgradeableClaimable {
 
     /**
      * @dev Get total amount borrowed for `borrower` from fixed term loans in USD
+     * Total borrowed amount would be 0 if no Fixed Term Loan is taken by the borrower at the moment
+     * And total loan amount + interest otherwise.
      * @param borrower Borrower to get amount borrowed for
      * @param decimals Precision to use when calculating total borrowed
      * @return Total amount borrowed for `borrower` in USD
      */
     function totalBorrowed(address borrower, uint8 decimals) public view returns (uint256) {
-        uint256 borrowSum;
+        address locker = borrowingMutex.locker(borrower);
+        if (!loanFactory.isLoanToken(IDebtToken(locker))) {
```

Comment:
This is a bit confusing in my opinion because `loanFactory` also has `isDebtToken()` function. How about casting `locker` to `ILoanToken2` instead?

Reply:
This will change after #1021 is merged